auth-web/src/composables/affiliations-factory.ts

```diff
@@ -27,18 +28,15 @@ export const useAffiliations = () => {
   const businesses = computed(() => businessStore.businesses)
   const headers: Ref<BaseTableHeaderI[]> = ref([])
 
-  /** Returns true if the affiliation is a Name Request. */
-  const isNameRequest = (business: Business): boolean => {
-    return (business.corpType?.code === CorpTypes.NAME_REQUEST && !!business.nameRequest)
-  }
+  /**
+   * @deprecated use isNameRequest from utils/Business
+   * */
+  const isNameRequest = BusinessUtils.isNameRequest
 
-  /** Returns true if the affiliation is a temporary business. */
-  const isTemporaryBusiness = (business: Business): boolean => {
-    return (
-      (business.corpType?.code || business.corpType) === CorpTypes.INCORPORATION_APPLICATION ||
-      (business.corpType?.code || business.corpType) === CorpTypes.REGISTRATION
-    )
-  }
+  /**
+   * @deprecated use isTemporaryBusiness from utils/Business
```

Comment:
No point in putting in deprecated, you're just creating tech debt.. I'd just replace them with your new call if possible

Reply:
I mean it is swapped, but any new code should use it from the BusinessUtils.
What I did here so I do not have to go and remove all the existing functions (and spend a week in testing), I just replaced it with reference to the new method.

This will let us if we need to do the change to any of the methods we can update them and cleaned them up at that time, and do focused test on specific method.
It also makes sure we are not having multiple places to manage the same thing.